test/CRUD.spec.js

```diff
@@ -75,6 +75,11 @@ describe("CRUD", () => {
       expect(crud.xhr.withArgs("/test/11", null, "DELETE").calledOnce).to.be
         .true;
     });
+
+    it("should call XHR from XHR with correct params and responseType", () => {
+      crud.xhr("/test",null,"GET",{responseType: 'blob'});
+      expect(crud.xhr.withArgs("/test", null, "GET", {responseType: 'blob'}).calledOnce).to.be.true;
```

Comment:
This test controls that:
1) We're calling xhr with args
2) That xhr was called with args

So this test should actually check that we're calling the axiosInstance method request, with args.

```
// First, create a stub out of the request method of the axios instance, with Sinon.
// Then we test, if calling the xhr method, the request method of axios instance was subsequently called.
expect(axiosInstance.request.withArgs("/test", null, "GET", {responseType: 'blob'}).calledOnce).to.be.true;
```